buildSrc/src/main/kotlin/com/datadog/gradle/plugin/jsonschema/generator/ClassJsonElementDeserializerGenerator.kt

```diff
@@ -27,17 +27,17 @@ class ClassDeserializerGenerator(
         val isConstantClass = definition.isConstantClass()
         val returnType = ClassName.bestGuess(definition.name)
 
-        val funBuilder = FunSpec.builder(Identifier.FUN_FROM_JSON)
+        val funBuilder = FunSpec.builder(Identifier.FUN_FROM_JSON_ELT)
```

Comment:
Should we call the function as `fromJsonObject` instead given that it takes `JsonObject` and not `JsonElement`?

`JsonElement` is an abstract class for `JsonArray`, `JsonNull`, `JsonPrimitive` and `JsonObject`.